AdyenDropIn/DropInComponentExtensions.swift

```diff
@@ -22,7 +22,10 @@ extension DropInComponent: PaymentMethodListComponentDelegate {
 
     internal func didLoad(_ paymentMethodListComponent: PaymentMethodListComponent) {
         let paymentMethodTypes = paymentMethods.regular.map(\.type.rawValue)
-        context.analyticsProvider.sendTelemetryEvent(flavor: .dropIn(paymentMethods: paymentMethodTypes))
+        let flavor = AnalyticsFlavor.dropIn(paymentMethods: paymentMethodTypes)
+        let amount = context.payment?.amount
+        let additionalFields = AdditionalAnalyticsFields(amount: amount, sessionId: AnalyticsForSession.sessionId)
+        context.analyticsProvider?.sendInitialAnalytics(with: flavor, additionalFields: additionalFields)
```

Comment:
Can we just make `DropInComponent` a `TrackableComponent ` and use default implementation?

Reply:
DropIn is not a `PresentableComponent` so the default extension won't work for it.